src/main/kotlin/com/chillibits/coronaaid/controller/v1/InfectedController.kt

```diff
@@ -36,10 +39,15 @@ class InfectedController {
         val infected = infectedRepository.findById(infectedId).orElseThrow { InfectedNotFoundException(infectedId) }
 
         // Check if already locked
-        if(infected.locked) throw InfectedLockedException(infectedId)
+        if(infected.locked) {
+            // Retrieve config record for 'autoResetOffset' (seconds -> conversion to millis)
+            val autoResetOffset = configRepository.findByConfigKey("autoResetOffset").configValue.toInt() * 1000
```

Comment:
We should add caching for this type of data in the future, but this is of course not part of this PR.

Reply:
Agreed.